common/scripted_triggers/wc_portrait_triggers.txt

```diff
@@ -472,6 +472,11 @@ potential_race_stone_tolvir_trigger = {
 	culture = { has_graphical_culture = stone_tolvir_portrait_gfx }
 	# NOT = { has_trait = creature_stone_tolvir }
 }
+}
+potential_race_drogbar_trigger = {
```

Comment:
Not added in
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114310672-952b2180-9afc-11eb-8201-3e8c7406703d.png)

</details>